docs/eventlisteners.md

```diff
@@ -417,6 +418,48 @@ spec:
         name: pipeline-template
 ```
 
+### BitBucket Interceptors
+
+BitBucket Interceptors contain logic to validate and filter webhooks that come from
+BitBucket. Supported features include validating webhooks actually came from BitBucket as well as
```

Comment:
"Bitbucket Server" or "Bitbucket Cloud"? It is two different products (and implementations) as what I know. Or both? This should be specified in the documentation.

Reply:
Both Bitbucket server and cloud provide Git repository management abilities.
Updated Doc accordingly 
Thank you